GetIntoTeachingApi/Services/Store.cs

```diff
@@ -118,6 +118,24 @@ public IQueryable<TeachingEvent> GetUpcomingTeachingEvents()
                 .Where(e => e.StartAt >= DateTime.UtcNow);
         }
 
+        public int GetNumberOfUnknownLocations()
+        {
+            IQueryable<Location> unknownLocations = GetUnknownLocations();

```

Comment:
Not a big deal but I tend to favour using `var` where possible as it makes refactoring easier:

```
var unknownLocations = GetUnknownLocations();
```

This would be fine as a one-liner as well:

```
GetUnknownLocations().Count()
```